docs/Using-Fleet/REST-API.md

```diff
@@ -5201,6 +5211,14 @@ Returns a list of all queries in the Fleet instance.
     "name": "osquery_schedule",
     "description": "Report performance stats for each file in the query schedule.",
     "query": "select name, interval, executions, output_size, wall_time, (user_time/executions) as avg_user_time, (system_time/executions) as avg_system_time, average_memory, last_executed from osquery_schedule;",
+    "team_id": null,
+    "frequency": 3600,
+    "platform": null,
+    "min_osquery_version": null,
```

Comment:
`platform` and `min_osquery_version` need changed.  They definitely shouldn't be `null`.